platform/api/aws/images.go

```diff
@@ -594,19 +594,21 @@ func (a *API) CopyImage(sourceImageID string, regions []string) (map[string]Imag
 			ch <- res
 		}()
 	}
-	wg.Wait()
-	close(ch)
+	go func() {
+		wg.Wait()
```

Comment:
With the `wg.Wait` inside of a goroutine doesn't the other logic need to change to accommodate the fact that it's potentially being ran before the goroutine is done processing?

Reply:
Not sure which bit you're referring to as "the other logic"... I'm guessing you meant the `range ch` below? That will keep looping until we close the channel, right? And that'll only happen once we've waited for all the goroutines.